### ISSUE TITLE
I would love to have a follow-up to the current active block or note placed in the folder/file window.

### ISSUE THREAD

**Issue by paranjay245** [iiz00/obsidian-multiple-notes-outline#12]
Hey, I am in love with the Multiple Notes Outline Plugin with File & Folder view. However, I would love to see a follow-up to the notes while we scroll, or to see the active file, heading, or bullet side in the bar of the window. It might be too much to ask, but I love the way it works. Anyways, I have seen the same feature with the Quiet Outline plugin, which has the same functionality Which I am expecting for.

Sorry if I am providing a lack of information. It is a weak or vague explanation. Also, thank you so much for the awesome plugin! :)

**Comment by iiz00** [comment]
Thank you for using the plugin!
The display of the active file (perhaps with a title or background color change)  was one of the features I wanted to implement eventually!
I hadn't thought about displaying the outline element at the cursor position, but I will look into it to see if it is possible.
I will also look at the plugin you mentioned as an example. I hope you can wait a while.

**Comment by iiz00** [comment]
Reading it again, I am not sure I understand your intent correctly.
I thought your FR was something like "visually highlight the currently active note or outline element at the cursor position in MNO view", but if I am wrong, I would appreciate more details.

**Comment by paranjay245** [comment]
> Reading it again, I am not sure I understand your intent correctly. I thought your FR was something like "visually highlight the currently active note or outline element at the cursor position in MNO view", but if I am wrong, I would appreciate more details.

Yes, exactly what I meant. I would like it to be positioned where I am currently located with text cursor. I don't mind if the mouse cursor position is not available, and it is not required for the purpose. We already have a quiet outline for the mouse cursor position, which could become janky/laggy with long documents. However, the keyboard/text cursor is the only thing required. The mouse cursor might be too much for huge documents, but it would be amazing to have it placed right where our keyboard is positioned. It would be truly mind-boggling.

Summary & Clarification: I would love to have a keyboard and text cursor, and the mouse cursor is not required, There is already a quiet outline for mouse cursor.

Sorry for the vague content and confusion. ❤️

**Comment by iiz00** [comment]
Thank you for the clarification 😄 
I would like to visualize the active file, but I'm not sure if I can visualize the outline element at the cursor position... I feel that there are performance issues with constantly monitoring the cursor movement and rewriting the view. I would like to try visualizing the active file first.
